packages/components/card/src/card.tsx

```diff
@@ -44,23 +66,74 @@ const Card = (props: TCardProps) => (
       background: ${props.theme === 'dark'
         ? designTokens.colorNeutral95
         : designTokens.colorSurface};
-    `}
-    // Allow to override the styles by passing a `className` prop.
-    // Custom styles can also be passed using the `css` prop from emotion.
-    // https://emotion.sh/docs/css-prop#style-precedence
-    className={props.className}
-  >
-    {props.insetScale === 'none' ? (
-      // Use a `<div>` to ensure that there is always a wrapper container.
-      // This is mostly useful in case custom styles are targeting this element.
+      cursor: ${props.isDisabled
+        ? 'not-allowed'
+        : isClickable
+        ? 'pointer'
+        : 'default'};
+      pointer-events: ${props.isDisabled ? 'none' : 'auto'};
+      :hover {
+        background: ${props.theme === 'dark'
+          ? isClickable
+            ? designTokens.colorNeutral90
+            : undefined
+          : isClickable
+          ? designTokens.colorNeutral98
+          : undefined};
+      }
+      // Disables link text styling
+      color: inherit;
+      // Change the opacity of the content, not the card itself
+      & > div {
+        opacity: ${props.isDisabled ? 0.5 : 1};
+      }
+    `,
+    className: props.className,
+  };
+
+  const content =
+    props.insetScale === 'none' ? (
       <div>{props.children}</div>
     ) : (
       <Inset scale={props.insetScale} height="expanded">
         {props.children}
       </Inset>
-    )}
-  </div>
-);
+    );
+
+  if (props.to) {
+    if (props.isExternalLink) {
+      if (typeof props.to !== 'string') {
+        throw new Error(
+          '`to` must be a `string` when `isExternal` is provided.'
+        );
+      }
+
+      return (
+        <a
+          {...commonProps}
+          role="link"
```

Comment:
We don't need the role as this is already the native one for an `a` element.

Reply:
Today I learned! Thank you, see 1cd4ef896fc51aeb43ee49dc59e4c5d7caf46cee